src/pages/individual/Individual.jsx

```diff
@@ -58,6 +58,36 @@ export default function Individual() {
     [queryClient, id],
   );
 
+  const individualDataForFeaturedPhoto = useMemo(
+    () => {
+      const allAssets = reduce(
+        individualData?.encounters,
+        (memo, encounter) => {
+          const newAssets = map(
+            get(encounter, 'annotations', []),
+            annotation => ({
+              src: annotation?.asset_src,
+              guid: annotation?.asset_guid,
+              filename: annotation?.created
+                ? 'Annotation created ' + annotation?.created
+                : 'Annotation with unknown creation date',
+            }),
+          );
+          return [...memo, ...newAssets];
+        },
+        [],
+      );
+      const assets = uniqBy(allAssets, asset => asset.src);
+      return {
+        assets: assets,
```

Comment:
You can just do `assets,` and it will work the same way

Reply:
FeaturedPhotos or whatever is looking for get(data, 'assets', [])... how would that work the same way?

Reply:
What does "just do assets" in this case mean?

Reply:
`{ assets: assets }` can be written as `{ assets }`. It is just a shorthand.

Reply:
I could have sworn that I tried it this way. I will confirm. Thanks.

Reply:
Done